integration/tests/conftest.py

```diff
@@ -196,13 +220,29 @@ def erc20_spl(
 
 
 @pytest.fixture(scope="session")
-def erc20_simple(web3_client_session, faucet):
-    erc20 = ERC20(web3_client_session, faucet)
-    return erc20
+def erc20_simple(web3_client_session, 
+                 faucet, 
+                 accounts_session, 
+                 pytestconfig: Config, 
+                 eth_bank_account
+):
+    network = pytestconfig.getoption("--network")
+    if network == "mainnet":
```

Comment:
you can do it without conditions, always with owner=accounts_session[0, for all networks

Reply:
fixed